source/features/latest-tag-button.tsx

```diff
@@ -118,14 +120,16 @@ async function init(): Promise<false | void> {
 	}
 
 	const defaultBranch = await getDefaultBranch();
+	const compareLink = (
+		<a className="btn btn-sm btn-outline tooltipped tooltipped-ne" href={`/${getRepoURL()}/compare/${latestTag}...${defaultBranch}`}>
+			<DiffIcon/>
+		</a>
+	);
 	if (currentBranch === defaultBranch) {
-		const aheadBy = await getAheadByCount(latestTag);
-		if (!aheadBy) {
-			return;
-		}
```

Comment:
We still need this condition and early return somewhere, probably right before: `const defaultBranch = await getDefaultBranch`

Reply:
That means you dont want to show 20+? Dont show it at all?

Reply:
The condition needs to be `if aheadBy === 0, return`

If HEAD is tagged, there's no extra info to show.

Reply:
isnt that handled here https://github.com/sindresorhus/refined-github/pull/3199/files#diff-5664f505bec7f184c9588cc203936104R109

```ts
	if (currentBranch === latestTag || aheadBy === 0) {
		link.setAttribute('aria-label', 'You’re on the latest release');
		link.classList.add('disabled', 'tooltipped', 'tooltipped-ne');
		return;
	}
```

Reply:
But I see that this condition already exists. All good.